src/foundry/common/documents/scene.mjs.d.ts

```diff
@@ -0,0 +1,372 @@
+     * @defaultValue `false`
+     */
+    active: fields.BooleanField;
+
+    /**
+     * Is this scene displayed in the top navigation bar?
+     * @defaultValue `true`
+     */
+    navigation: fields.BooleanField<{ initial: true }>;
+
+    /**
+     * The sorting order of this Scene in the navigation bar relative to siblings
+     * @defaultValue `0`
+     */
+    navOrder: fields.NumberField<{ required: true; nullable: false; integer: true; initial: 0 }>;
+
+    /**
+     * A string which overrides Scene name for display in the navigation bar
+     * @defaultValue `""`
+     */
+    navName: fields.HTMLField;
+
+    // Canvas Dimensions
+    /**
+     * An image or video file that provides the background texture for the scene.
+     * @defaultValue see {@link TextureData}
+     */
+    background: TextureData<{}, { categories: ["IMAGE", "VIDEO"]; initial: null; wildcard: false }>;
+
+    /**
+     * An image or video file path providing foreground media for the scene
+     * @defaultValue `null`
+     */
+    foreground: fields.FilePathField<{ categories: ["IMAGE", "VIDEO"] }>;
+
+    /**
+     * The elevation of the foreground layer where overhead tiles reside
+     * @defaultValue `null`
+     */
+    foregroundElevation: fields.NumberField<{ required: false; positive: true; integer: true }>;
+
+    /**
+     * A thumbnail image which depicts the scene at lower resolution
+     * @defaultValue `null`
+     */
+    thumb: fields.FilePathField<{ categories: ["IMAGE"] }>;
+
+    /**
+     * The width of the scene canvas, normally the width of the background media
+     * @defaultValue `4000`
+     */
+    width: fields.NumberField<{ integer: true; positive: true; initial: 4000 }>;
+
+    /**
+     * The height of the scene canvas, normally the height of the background media
+     * @defaultValue `3000`
+     */
+    height: fields.NumberField<{ integer: true; positive: true; initial: 3000 }>;
+
+    /**
+     * The proportion of canvas padding applied around the outside of the scene
+     * dimensions to provide additional buffer space
+     * @defaultValue `0.25`
+     */
+    padding: fields.NumberField<{ required: true; nullable: false; min: 0; max: 0.5; step: 0.05; initial: 0.25 }>;
+
+    /**
+     * The initial view coordinates for the scene
+     * @defaultValue `undefined`
+     */
+    initial: fields.SchemaField<{
+      x: fields.NumberField<{ integer: true; nullable: true; initial: undefined }>;
+      y: fields.NumberField<{ integer: true; nullable: true; initial: undefined }>;
+      scale: fields.NumberField<{ nullable: true; min: 0.25; max: 3; initial: undefined }>;
+    }>;
+
+    /**
+     * The color of the canvas displayed behind the scene background
+     * @defaultValue `"#999999"`
+     */
+    backgroundColor: fields.ColorField<{ initial: "#999999" }>;
+
+    // Grid Configuration
+    /**
+     * Grid configuration for the scene
+     * @defaultValue `CONST.GRID_TYPES.SQUARE`
```

Comment:
This doesn't seem to be the default value for the entire property. That's only the default for the `type` sub-property. Please also document the default values for the sub properties on the properties themselves.